README.md

```diff
@@ -49,7 +49,7 @@ First of all, please have a look at the [Ansible module development](https://doc
 When looking at actual modules in this repository ([`foreman_domain`](plugins/modules/foreman_domain.py) is a nice short example), you will notice a few differences to a "regular" Ansible module:
 
 * Instead of `AnsibleModule`, we use `ForemanEntityApypieAnsibleModule` (and a few others, see [`plugins/module_utils/foreman_helper.py`](plugins/module_utils/foreman_helper.py)) which provides an abstraction layer for talking with the Foreman API
-* We provide a `name_map` that translates between Ansible module parameters and Foreman API parameters, as nobody wants to write `organization_ids` in their playbook when they can write `organizations`
+* Instead of `argument_spec`, we provide an enhanced version called `entity_spec`. It handles the translation from module parameters to Foreman API parameters, as nobody wants to write `organization_ids` in their playbook when they can write `organizations`
```

Comment:
```suggestion
* Instead of Ansible's `argument_spec`, we provide an enhanced version called `entity_spec`. It handles the translation from Ansible module arguments to Foreman API parameters, as nobody wants to write `organization_ids` in their playbook when they can write `organizations`
```